Vz cpec

Starnite opened the pull request:
@ericmjl The multiple sequence alignment was done with Clustal Multiple Sequence Alignment. I started writing code for stitching things together (cpec-translate.ipynb under the cpec folder under experiments), but it isn't working just yet.

Starnite committed:
Aligned cpec sequences with template

Starnite committed:
Translated and aligned full mutant PB2 protein sequences

ericmjl reviewed (changes requested):
Good stuff! I'm going to show you where I hope the bar is for "reproducible science", so bear with me here.

How was the multiple sequence alignment done? Do you have a script, or software parameters noted down somewhere?

The FASTA files have telltale signs of being hand-stitched together. Rather than that, I think it'd be worthwhile investing some time in drafting code that does the stitching for you instead.

Starnite committed:
stitch and translate code WIP

ericmjl commented:
@Starnite cool stuff! Next time, though, don't worry about editing your commit comment, but continue the conversation using the text box at the end.

Be sure to note in the report whether the alignment was done locally or via the EBI web server.

I took a quick look at the code, and it's looking not bad. When do you want to chat about it? I think I can guide you past what you've gotten done so far.

ericmjl commented:
@Starnite: just thought of one more thing here.

I think I made a mistake in letting you use EM-28 as the sequencing primer. It should actually be something ~100-120 b.p. upstream of the start of EM-28. In that way, the sequencing results will have good-quality sequencing traces at about the point that the PCR product begins. Likewise, I should have asked you to do a reverse sequencing reaction as well, with the reverse sequencing primer lying ~100-120 b.p. upstream (downstream in the opposite direction) of the start of EM-27. 

```

--sequencing-->          --cloning-->
5'-------------------------------------------------------------------------------------------3'
3'-------------------------------------------------------------------------------------------5'
                                                            <--cloning--        <--sequencing--
```

Starnite committed:
Added alignment method

Starnite committed:
Added link

Starnite committed:
updated

Starnite committed:
Stitched together sequences

Put in a dictionary for now, will create records tomorrow

Starnite committed:
Tried to translate

There are still some gaps present. There are a couple of deletions and
frameshifts in the sequences that I didn't consider before.

Starnite committed:
Aligned amino acid sequences

I still haven't installed ClustalW...

Starnite committed:
Implemented mutation counting

- I think it's time I downloaded something to calculate Liechtenstein
distances
- Still need to write to record
- L16 has a lot of mutations because of a frameshift - edit mutation
counting function to recognize deletions/insertions

Starnite committed:
Created records and saved to file

Starnite committed:
Deleted gaps, deleted mutation locations from file

Starnite committed:
Goals for next two weeks

Starnite committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into vz-cpec

Starnite committed:
Started cell culture for pol assay replicates

Starnite committed:
Gel + PCR re-run

I'll merge our two files after the cpec assembly is complete

Starnite committed:
Replicate pol assay updates

Starnite committed:
Updated transfection protocol

Starnite committed:
Added cell counting protocol

ericmjl commented on protocols/cell-culture.md:
```diff
@@ -16,14 +16,19 @@ Notes:
+**Counting cells**
+
+1. Withdraw small volume of cells from the 1mL suspension of cells in DMEM and dilute with dye, about 1:5.
+2. Count cells in four corners of grid in hemacytometer. Take average. Number of cells in 1mL = number of cells counted * dilution factor * 10E4.
```
Can you change 10E4 to `1 x 10<sup>4</sup>` for clearer formatting?

Starnite committed:
Today's seeding and counting

ericmjl commented:
ping @Starnite: your `goals/Oct-16.md` has a conflict, FYI.

Starnite committed:
Transfection Updates

Starnite committed:
Cell count in pol assay wells

Starnite committed:
Attempted luminescence updates

Starnite committed:
Replication luminescence results

Starnite committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into vz-cpec

# Conflicts:
#	goals/Oct-16.md

Starnite committed:
Fixed cell culture dilution order of magnitude error

Starnite committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into vz-cpec

Starnite committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into vz-cpec

Starnite committed:
Fixed exponent notation, made two ape files of PB2 for designing barcode insertion primers

I think we can merge this pull request now

ericmjl committed:
fixed conflict

ericmjl committed:
Merge branch 'master' into vz-cpec

ericmjl committed:
Merge branch 'vz-cpec'

ericmjl committed:
Merge branch 'master' into vz-cpec

ericmjl committed:
Merge remote-tracking branch 'origin/vz-cpec' into vz-cpec

ericmjl commented:
@Starnite: merged. Great work!
